examples/chef/devices/rootnode_roboticvacuumcleaner_1807ff0c49.matter

```diff
@@ -1417,6 +1677,22 @@ endpoint 1 {
     callback attribute clusterRevision;
   }
 
+  server cluster PowerSource {
```

Comment:
is power source required for RVC? The spec seems to require run-more and operational state only. Did we make this change on purpose? If so it should be described in the PR summary.

Reply:
Having power source is relevant since RVCs are battery operated